src/main/java/uk/gov/hmcts/reform/sscs/model/DescendantCaseData.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+
+import java.util.Map;
+
+@EqualsAndHashCode(callSuper = true)
+@Builder (builderMethodName = "DescendantCaseDataBuilder")
+public class DescendantCaseData extends uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData {
```

Comment:
This still needs changing

Reply:
Echoing Del - not sure why this class is needed